### ISSUE TITLE
WMS Time Capabilities: adapt timerange

### ISSUE THREAD

**Issue by JakobMiksch** [meggsimum/sauber-sdi-docker#125]
Currently the time range for the WMS Time ends at `2021-06-29` - see [Capabilities](https://sauber-sdi.meggsimum.de/geoserver/image_mosaics/wms?service=WMS&version=1.3.0&request=GetCapabilities).

```xml
<Dimension name="time" default="2021-06-29T23:00:00Z" units="ISO8601" nearestValue="1">
2020-01-01T00:00:00.000Z/2021-06-29T23:00:00.000Z/PT1H
</Dimension>
```

The time range should be extended to the last valid time.

**Comment by JakobMiksch** [comment]
This seems to work meanwhile

@jdk24 can we close this or does it still depend on an unmerged PR?

**Comment by jdk24** [comment]
This was the issue with GeoServer ImgMosaic data being created in a different spots, depending on it being created via REST or the GUI. 

The Raster downloader couldn't find the path and thus didn't insert the metadata correctly. 

This was temporarily altered on the server for us to check, and will be permanently fixed by #113, I added a reference there.
